res/menu/conversation_clear.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:title="@string/menu_delete_messages"
```

Comment:
I reused this string but maybe a "clear chat"/"clear history" string could be added